### ISSUE TITLE
Truecolor support

### ISSUE THREAD

**Issue by gerion0** [cslarsen/jp2a#10]
I noticed that the tool generates the whole color palette when generating HTML but seems restricted to 16(?) colors when generating ANSI codes.

Modern terminals also support truecolor. It would be nice, if jp2a could generate text for them or even better if there is a flag to control the amount of colors to output (also for HTML). [Here](https://gist.github.com/XVilka/8346728) I found a good summary of truecolor support for terminals.

**Comment by cslarsen** [comment]
Definitely, but I'm not actively working on jp2a at the moment. Setting the "help wanted" label.

**Comment by IllyaMoskvin** [comment]
I used jp2a to generate truecolor output for terminals by making it output HTML and transforming that output using `sed` and [bash string manipulation](https://www.tldp.org/LDP/abs/html/string-manipulation.html). Check out [aic.sh#L358-L474](https://github.com/art-institute-of-chicago/aic-bash/blob/ce1dfb70d66e2a339a4ab058eaa1ec94e84944e3/aic.sh#L358-L474) from here:

https://github.com/art-institute-of-chicago/aic-bash

![example](https://user-images.githubusercontent.com/3881840/71372609-7e189700-257a-11ea-8be6-b167394b9f41.png)
